packages/cli/src/commands/account/new.ts

```diff
@@ -38,13 +39,23 @@ export default class NewAccount extends LocalCommand {
       description:
         "Language for the mnemonic words. **WARNING**, some hardware wallets don't support other languages",
     }),
```

Comment:
I would much prefer the mnemonic flag (and password) to pass a file that contains the mnemonic rather than have that exposed on the command line (hence in the ps list and in bash history)

Reply:
Yes, I totally agree
In two different plain files? (mnemonic and password)
Or have a file with a configuration json?

Reply:
The only problems that I find on using one file, is that we are adding a possible error in how the user used the command
if for some reason added the password behind instead of the front, of if it's using a mnemonic of other strength, for example 21, and with a password of 3 words separated by a whitespace (unlikely, of course, but I prefer to go to the safe options of two files)

Reply:
You could also just have the mnemonic as-is in the file, and pop a silent prompt where the user inputs their password to stdin (without showing the password in the console of course)